test/830_cni_plugin_test.sh

```diff
@@ -85,6 +113,24 @@ assert_raises "[ $C2IP != $C3IP ]"
 assert_raises "[ $BRIP != $C3IP ]"
 assert_raises "exec_on $HOST1 c1 $PING $C3IP"
 
+# get the value of hairpin on the veth for each of ch0, ch1, ch2
+CH0_PEER_ID=$(exec_on $HOST1 ch0 cat /sys/class/net/eth0/iflink)
+CH1_PEER_ID=$(exec_on $HOST1 ch1 cat /sys/class/net/eth0/iflink)
+CH2_PEER_ID=$(exec_on $HOST1 ch2 cat /sys/class/net/eth0/iflink)
+
+CH0_PEER=$(run_on $HOST1 ip link show | awk "/^${CH0_PEER_ID}:/"' {print $2}' | sed 's/@.*$//g' )
```

Comment:
I was wondering how the naming worked. I can live with that fragility if you can. I like going into the container and getting the peer interface number better, but, yeah, can live with this. Maybe we should at least leave a comment in case future changes break it, save them the headache of digging.